examples/step-57/doc/results.dox

```diff
@@ -0,0 +1,429 @@
+    <th>&nbsp;    </th>
+  </tr>
+
+  <tr>
+    <th>7         </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+    <th>6.42323e-15 </th>
+    <th>9         </th>
+    <th>&nbsp;</th>
+    <th>&nbsp;         </th>
+    <th>&nbsp; </th>
+    <th>&nbsp;         </th>
+    <th>&nbsp;    </th>
+    <th>&nbsp;    </th>
+  </tr>
+</table>
+The sequence of generated grids looks like this:
+<TABLE ALIGN="center">
+  <tr>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Mesh0.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Mesh1.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Mesh2.png" width="232px" alt="">
+    </td>
+  </tr>
+  <tr>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Mesh3.png" width="232px" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Mesh4.png" width="232px" alt="">
+    </td>
+  </tr>
+</table>
+We compare the solution by Newton's method with reference solution from [5].
+<img src="https://www.dealii.org/images/steps/developer/step-57.compare-Re7500.svg" style="width:50%" alt="">
+The following picture presents the graphical result.
+<img src="https://www.dealii.org/images/steps/developer/step-57.Re7500_Streamline.png" alt="">
+
+Furthermore, the error consists of the nonlinear error,
+which is decreasing as Newton's iteration going on, and the discrete error, 
+which depends on the mesh size. That is why we have to refine the
+mesh and repeat Newton's iteration on the next finer mesh. From table above, we can
+see that the residual (nonlinear error) is below 1e-12 on each mesh, but the
+following picture shows us the difference between solutions on subsequently finer 
+meshes.
+
+<img src="https://www.dealii.org/images/steps/developer/step-57.converge-Re7500.svg" style="width:50%" alt="">
+
+<a name="extensions"></a>
+
+<h3>Possibilities for extensions</h3>
+
+<h4>Compare to other solvers</h4>
+
+It is easy to compare the currently implemented linear solver to just using
+UMFPACK for the whole linear system. You need to remove the nullspace
+containing the constant pressures as it is done in step-56. More interesting
+is the comparison to other state of the art preconditioners like PCD. It turns
+out that the preconditioner here is very competitive, as can be seen in the
+paper [2].
+
+The following table shows the timing results between our iterative approach
+ (FGMRES) compared to a direct solver (UMFPACK) for the whole system
+with viscosity set to 1/400. Even though we use the same direct solver for
+the velocity block in the iterative solver, it is considerably faster and
+consumes less memory. This will be even more pronounced in 3d.
```

Comment:
This is actually quite interesting and has occurred to me recently about step 22. The A block of the matrix is by far the largest part of the overall matrix. Why is it that using a direct solver for it takes significantly less memory than for the whole matrix? Because it has much smaller coupling?